support/pom.xml

```diff
@@ -38,13 +38,6 @@
       <optional>true</optional>
     </dependency>
 
-    <dependency>
```

Comment:
have you verified / is there a test that shows that the private key isn't exported in plain text?

Reply:
YES :laughing: 

Arguably this should be updated to check the direct key
https://github.com/jenkinsci/configuration-as-code-plugin/blob/40c83890bfb3f499c97a69be52a69bad1e0427c3/integrations/src/test/java/io/jenkins/plugins/casc/CredentialsTest.java#L32-L36

https://github.com/jenkinsci/configuration-as-code-plugin/blob/40c83890bfb3f499c97a69be52a69bad1e0427c3/integrations/src/test/resources/io/jenkins/plugins/casc/GlobalCredentials.yml#L19-L21

Reply:
apparently not ready, see CI... the test ran locally :confused:

Reply:
@jvz we need some more magic it seems!

Reply:
It really needs to be, as it was a security issue, and we don't want it accidentally re-introduced

Reply:
it works locally, I don't get it :confused:

Reply:
I'm confused why its trying to use this:
```
Caused by: java.lang.IllegalArgumentException: No com.cloudbees.jenkins.plugins.sshcredentials.impl.BasicSSHUserPrivateKey$PrivateKeySource implementation found for FileOnMasterPrivateKeySource
Possible solution: Try to install 'configuration-as-code-support' plugin
```

it should be directEntry not the fileOnMaster...

Reply:
Fixed in: https://github.com/jenkinsci/configuration-as-code-plugin/pull/862/commits/202ab7b2ff378c0a2ddb614b72a3d497d25b380c